### PR TITLE
ledger-report-toggle-default-commodity: New command

### PR DESCRIPTION
**Pull request by jabranham**
Bound to "$" in ledger-report-mode-map

### CODE REVIEW

File: ledger-report.el [hunk 1]
```diff
@@ -184,6 +187,7 @@ when running reports?"
       #'ledger-report-edit-report)
     (define-key map (kbd "M-p") #'ledger-report-previous-month)
     (define-key map (kbd "M-n") #'ledger-report-next-month)
+    (define-key map [?$] #'ledger-report-toggle-default-commodity)
```
Comment (purcell) [inline ledger-report.el]: For consistency, can I suggest:

```suggestion
    (define-key map (kbd "$") #'ledger-report-toggle-default-commodity)
```

File: ledger-report.el [hunk 2]
```diff
@@ -643,6 +647,20 @@ arguments returned by `ledger-report--compute-extra-args'."
   (interactive)
   (ledger-report--change-month 1))
 
+(defun ledger-report-toggle-default-commodity ()
+  "Add or remove \"--exchange `ledger-reconcile-default-commodity' to the current report."
+  (interactive)
+  (unless (derived-mode-p 'ledger-report-mode)
+    (user-error "Not a ledger report buffer"))
+  (if (string-match
```
Comment (purcell) [inline ledger-report.el]: Here you probably want `string-match-p`, which avoids setting `match-string` etc. as a side-effect.

File: ledger-report.el (hunk 1 above)
Comment (jabranham) [reply ledger-report.el]: Ok, changed.

File: ledger-report.el (hunk 2 above)
Comment (jabranham) [reply ledger-report.el]: Done, thanks

**Comment by jabranham** [comment]
Thanks @purcell I made those changes you recommended, rebased off master, and pushed here.

**Comment by purcell** [comment]
Awesome, thanks!

**Pull request merged**